ts/Data/Stores/DataStore.ts

```diff
@@ -342,6 +326,20 @@ abstract class DataStore implements DataEvent.Emitter {
         return addEvent(this, type, callback);
     }
 
+    /**
+     * The default save method, which fires the `afterSave` event
+     *
+     * @return {Promise<DataStore>}
+     * The saved store.
+     *
+     * @emits DataStore#afterSave
+     * @emits DataStore#saveError
+     */
+    public save(): DataPromise<this> {
+        fireEvent(this, 'saveError', { table: this.table });
```

Comment:
Why do we use saveError instead of `afterSave` event?

Reply:
There is no implementation for it in the basis class. Store implementations have to overwrite it to support it without error.